### PR TITLE
LDAP TLS Options addition to documentation

### PR DESCRIPTION
**Pull request by pke11y**
### Fixes: #523 

Add TLS Options section to set SSL settings. The changes could be added to the The General Server Configuration section but then that assumes TLS is the default. Keeping it separate should be cleaner.

### CODE REVIEW

File: nautobot/docs/configuration/authentication/ldap.md [hunk 1]
```diff
@@ -84,8 +84,21 @@ AUTH_LDAP_BIND_PASSWORD = "demo"
 LDAP_IGNORE_CERT_ERRORS = True
 ```
 
+### TLS Options
+
 STARTTLS can be configured by setting `AUTH_LDAP_START_TLS = True` and using the `ldap://` URI scheme.
 
+Apply TLS settings to the internal SSL context on nautobot by configuring `ldap.OPT_X_TLS_NEWCTX` with value `0`. 
+
+```
```
Comment (jathanism) [inline nautobot/docs/configuration/authentication/ldap.md:93]: It's not clear to me here... Is `AUTH_LDAP_START_TLS = True` also required for this to work? If so, would you mind including it in the example config you provided, along w/ the `ldap://` URI for the service?

**Review by jathanism — APPROVED** [review]
Just the one comment, but good to go!! Thank you!

File: nautobot/docs/configuration/authentication/ldap.md (hunk 1 above)
Comment (pke11y) [reply nautobot/docs/configuration/authentication/ldap.md:93]: Yes will include the commands. TLS can be configured as described or using `ldaps://` URI and `AUTH_LDAP_START_TLS = False`, which is a little less intuitive.

**Comment by jathanism** [comment]
@pke11y Can you please merge in the latest from `develop` branch and push it up so that the tests will succeed? Thanks!

**Comment by pke11y** [comment]
@jathanism merged from develop as requested.

**Pull request merged**